android/app/build.gradle

```diff
@@ -117,7 +117,6 @@ def getVersionName = { ->
 
 android {
     compileSdkVersion 24
-    buildToolsVersion "26.0.2"
```

Comment:
What is the potential downside here? Looks like there was a reason for this difference in version?

Reply:
It's not essential to remove this as 26.0.2 is already the minimum supported version for Android Gradle Plugin 3.0.1. But there's no longer a need to specify it explicitly as the gradle plugin now specifies a default buildToolsVersion. Felt it was better to go with that, what do you think?

For the upstream projects these lines must be removed by updateUnforkedNodeModules.sh else we get errors like
```
WARNING: The specified Android SDK Build Tools version (25.0.3) is ignored, as it is below the minimum supported version (26.0.2) for Android Gradle Plugin 3.0.1.
Android SDK Build Tools 26.0.2 will be used.
To suppress this warning, remove "buildToolsVersion '25.0.3'" from your build.gradle file, as each version of the Android Gradle Plugin now has a default version of the build tools.
```